x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/SparseVectorFieldMapper.java

```diff
@@ -104,6 +104,7 @@ public DocValueFormat docValueFormat(String format, ZoneId timeZone) {
 
         @Override
         public Query existsQuery(QueryShardContext context) {
+            //TODO how can this work if this field never has doc_values?
```

Comment:
Same as above

Reply:
this is a bit tricky because the field is no longer supported, fielddataBuilder was removed yet there are doc_values in the index. I will figure it out

Reply:
This is addressed by #62646